samples/qa-a11y/src/views/Heading.js

```diff
@@ -0,0 +1,22 @@
+import Heading from '@enact/agate/Heading';
+import React from 'react';
+
+import Section from '../components/Section';
+
+import appCss from '../App/App.module.less';
+
+const HeadingView = () => (
+	<>
+		<Section title="Default">
+			<Heading alt="Normal" size="large" showLine>Text 0</Heading>
```

Comment:
Does Heading itself could get a focus?
Do you need help from other component like you did in BodyText?

Reply:
Fixed like BodyText.